### DISCUSSION TITLE
Customize Home Page option

### DISCUSSION THREAD

**Discussion by Bottlecapps** [apache/superset#18571] [Ideas]
In our business case we provide access to our Superset Dashboard to external customers. When they login it takes them to this page
![image](https://user-images.githubusercontent.com/17992090/152361228-b334d598-8704-4b89-bea0-34d90c8b2f07.png)

The users get totally confused on what to do next. My suggestion is to allow us to configure the default home page based on the user role. So in this case I would like my users to be able to directly land on the Dashboard page. Or maybe have them go directly into a dashboard view.

![image](https://user-images.githubusercontent.com/17992090/152361495-59a1d820-c2c7-42c1-bf72-9d9a9d124f05.png)

**Comment by CFM90** [comment]
Hello, 

I found a Solution on Stack overflow working for. You need to add the provided code to your superset config and it redirects do dashboard/list. 

Here is the [Stackoverflow Link](https://stackoverflow.com/questions/57357488/how-can-i-modify-the-url-of-the-welcome-page-superset)

**Comment by JohnDietrich-Pepper** [comment]
We have a similar use case.  We would really like to setup some sort of folders or similar to organize the dashboards and have only that structure appear.  While the stackoverflow link is cool it would be nice to have out of the box functionality to ensure new users directly see the dashboard pages.

**Comment by santosh-sahoo334** [comment]
You can try out this is `config.py `
https://gist.github.com/d9k/a638071ce7146ef01c27779a51d96f2b
You can re-direct the User directly to Dashboard without displaying the `/dashboard/list`

**Reply by dbaltor** [reply]
I've tested this answer from the thread above suggested by @santosh-sahoo334 to configure Superset 4 and it works just fine. I have configured the landing page to point to the **Dashboards** page
https://gist.github.com/d9k/a638071ce7146ef01c27779a51d96f2b?permalink_comment_id=4770788#gistcomment-4770788
